src/main/java/core/basesyntax/exception/LoginValidateException.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax.exception;
+
+public class LoginValidateException extends RuntimeException {
```

Comment:
same as with previus comment about the class name of the exception, class names should be nouns like here:
```suggestion
public class InvalidLoginException extends RuntimeException {
```

Reply:
not resolved